packages/landingpage/components/getting-started/layout.tsx

```diff
@@ -21,17 +22,42 @@ const SANDBOX_MAP: { [key in Framework]?: string } = {
 };
 
 const Layout = ({ children, framework, sandboxUrl }: Props) => {
-  const { push } = useRouter();
+  const router = useRouter();
   const frameworkName = NameByFramework[framework];
 
+  useEffect(() => {
+    const handlePageLoad = () => {
+      if (router.asPath.includes('#')) {
+        const hash = window.location.hash.substring(1);
+        const targetElement = document.getElementById(hash);
+        if (targetElement) {
+          const headerOffset = 80;
+          const elementPosition = targetElement.getBoundingClientRect().top;
+          const offsetPosition =
+            elementPosition + window.pageYOffset - headerOffset;
+
```

Comment:
You could get the hight of the navigation programmatically with `offsetHeight()` and set it as the `headerOffset`. If we decide to change the nav-hight in the future, we won't have to manually change it here.

Reply:
Good Idea! thanks for the review :)